typeDefs.js

```diff
@@ -333,6 +333,10 @@ const typeDefs = gql`
     available: Boolean
     additionalPrice: [CurrencySetInput]
   }
+  input UserRateInput {
+    user: ID!
```

Comment:
We don't need to have `user`, `userId` should be taken from the JWT token of authed user

Reply:
removed